client/src/components/JobInformation/CodeRow.vue

```diff
@@ -9,7 +9,7 @@
         </td>
         <td v-if="codeItem">
             <b-row align-v="center">
-                <b-col cols="9">
```

Comment:
Instead of a specific number of cols at all, should we just make the second column text not break and the first column take all other available space?

Reply:
> Instead of a specific number of cols at all, should we just make the second column text not break and the first column take all other available space?

sure, what's the option for that?

Reply:
I think the right fix would be to use flex and not grid layout, but I'm honestly happy enough with the latest revision with the very small toggle that I don't mind one way or another.